### DISCUSSION TITLE
two ebud adapter with one gas bolier (ebus adapter 3 + ebus adapter 5)

### DISCUSSION THREAD

**Discussion by wdowa79** [john30/ebusd#1232] [Q&A]
Hi, 

Now I have vaillant ecoTec plus VC PL 186/3-5 R3 + ebus adapter 3.
Can i connect to this configuration another ebus adapter v5? 
need i specific configuration of ebus adapter?

Peter

**Comment by Koky05** [comment]
You can connect as many adapters as you wish but for another adapters you need to specify different address because all by default work on adress 31.

`-a, --address=ADDR
Use hex ADDR as own master bus address [31]`
